frontend/src/stimulus/controllers/dynamic/project-storage-form.controller.ts

```diff
@@ -101,19 +100,22 @@ export default class ProjectStorageFormController extends Controller {
       storage: this.storage,
     };
 
-    this.modalService
-      .pipe(
-        switchMap((service) => service.show(LocationPickerModalComponent, 'global', locals, false, false, this.OutletTarget)),
```

Comment:
The `OutletTarget` is needed and can't simply be removed. It is responsible for rendering the Angular Dialog as part of the topLayer that the Primer Dialog creates. Since that is gone, the modal is rendered behind the dialog which is what @NobodysNightmare observed in his screenshot.